src/main/java/net/greatstart/model/Project.java

```diff
@@ -1,15 +1,33 @@
 package net.greatstart.model;
 
-import java.time.LocalDate;
+import org.springframework.stereotype.Component;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.persistence.Embedded;
+import javax.persistence.ManyToOne;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import javax.persistence.FetchType;
 import java.util.List;
 
-public class Project {
-    private boolean isVerifed;
-    private String name;
+@Component
+@Entity
+@Table(name = "projects")
+public class Project extends AbstractModel {
+    @Embedded
     private ProjectDescription desc;
-    private int cost;
-    private int minInvestment;
-    private LocalDate addDate;
+
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "owner_id")
     private User owner;
-    private List<User> investors;
+
+    /*
+     * Does not exist in DB table
+     * private List<User> investors;

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)